packages/addons-website/scss/components/_type-weight.css

```diff
@@ -0,0 +1,48 @@
+@import '../prefix';
+
+/* .#{$prefix}--type-weight { */
+    /* background: $ibm-colors__white;
+    padding: $spacing-md;
+    margin-bottom: rem(24px);
+  
+    p {
+      font-size: rem(48px);
+      padding: 0;
+      line-height: 1.25;
+      color: $ibm-colors__gray-80;
+    }
+   */
+    .#{$prefix}--ibm-type-italic {
```

Comment:
we should be able to import these class names from `@import '@carbon/type/scss/classes';`

also, was this supposed to be a css file or scss?